projectgenerator/tests/test_dump.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+"""
+/***************************************************************************
+    begin                :    24.5.2018
+    git sha              :    :%H$
+    copyright            :    (C) 2018 Matthias Kuhn
+    email                :    matthias@opengis.ch
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+"""
+
+import nose2
+
+from projectgenerator.tests.utils import testdata_path
+from projectgenerator.libqgsprojectgen.generator.generator import Generator
+from qgis.testing import unittest, start_app
+
+from subprocess import call
+import os
+
+start_app()
+
+
+class TestCustomDump(unittest.TestCase):
+
+    def test_ili2pg_dump_without_metattr(self):
+        myenv = os.environ.copy()
+        myenv['PGPASSWORD'] = 'docker'
+        call(["pg_restore", "-Fc", "-hpostgres", "-Udocker", "-dgis", testdata_path("dumps/_nupla_dump")], env=myenv)
```

Comment:
I always make spaces between parameters, like `-h postgres` but I guess this works as well.

Reply:
With `pg_restore` also? I normally also make those spaces, but had trouble here.

Reply:
Also on postgresql.org/docs there is the example for `pg_restore` with spaces. I don't know where the problems come from, but I suppose when it works here without them, then it's fine as well.

Reply:
I also had to use an environment variable for the password, might be related to an old version of pg_restore. I'll keep this working approach and merge.

Thanks for the review